### ISSUE TITLE
Does triton support Keras option during inference training=True

### ISSUE THREAD

**Issue by vencivat** [triton-inference-server/server#6107]
**Is your feature request related to a problem? Please describe.**
Some neural network layers behave differently during training and inference, for example Dropout and BatchNormalization layers. For example BatchNormalization layer:

During training , the layer normalizes its output using the mean and standard deviation of the current batch of inputs
During inference, the layer normalizes its output using a moving average of the mean and standard deviation of the batches it has seen during training

The training argument lets the layer know which of the two "paths" it should take. If you set this incorrectly, your network might not behave as expected.

I need to understand if I can pass training=True when doing inference with triton  tensorflow backend

**Describe the solution you'd like**
I would like to know if it is possible to run inference in training mode for triton tensorflow backend, 
possible with config flag in config.pbtxt

**Comment by jbkyang-nvi** [comment]
I believe we don't allow this if the way to do it is by passing `training=true` when calling `Infer`. Correct me if I'm wrong @tanmayv25

**Comment by dyastremsky** [comment]
Closing due to inactivity. Please let us know if you would like to reopen this issue for follow-up.
